src/commons/controlBar/ControlBarShareButton.tsx

```diff
@@ -66,12 +89,36 @@ export class ControlBarShareButton extends React.PureComponent<ControlBarShareBu
                   onChange={this.handleChange}
                   style={{ width: 175 }}
                 />
+                <>{console.log(this.props.programConfig)}</>
```

Comment:
Please remove the console.log, also please don't put JS code inside JSX.

Reply:
removed.